docs/src/Typeahead.doc.js

```diff
@@ -0,0 +1,196 @@
+        href: 'basicExample',
+      },
+      {
+        name: 'defaultItem',
+        type: '{ label: string, value: string }',
+        description: 'The default item set in the Typeahead',
+        required: false,
+        href: 'defaultItemExample',
+      },
+      {
+        name: 'id',
+        type: 'string',
+        required: true,
+        href: 'basicExample',
+      },
+      {
+        name: 'label',
+        type: 'string',
+      },
+      {
+        name: 'noResultText',
+        type: 'string',
+        href: 'basicExample',
+        required: true,
+        description: 'The text shown when the input value returns no matches',
+      },
+      {
+        name: 'onBlur',
+        type:
+          '({ event: SyntheticFocusEvent<HTMLInputElement> | SyntheticEvent<HTMLInputElement> , value: string }) => void',
+        required: false,
+        description: 'Callback when you focus outside the component ',
+        href: 'basicExample',
+      },
+
+      {
+        name: 'onChange',
+        type: '({ event: SyntheticInputEvent<>, value: string }) => void',
+        required: false,
+        description: 'Callback when user types into the control input field',
+        href: 'basicExample',
+      },
+
+      {
+        name: 'onFocus',
+        type: '({ event: SyntheticFocusEvent<>, value: string }) => void',
+        required: false,
+        description: 'Callback when you focus on the component',
+        href: 'basicExample',
+      },
+      {
+        name: 'onSelect',
+        type: '({ event: SyntheticInputEvent<>, item: Object }) => void',
+        required: false,
+        description: 'Callback when you select an item ',
+        href: 'basicExample',
+      },
+      {
+        name: 'placeholder',
+        type: 'string',
+        href: 'basicExample',
+      },
+      {
+        name: 'searchField',
+        type: 'string',
+        description:
+          'The property name on which you want to be able to search on',
+        href: 'basicExample',
+        defaultValue: 'label',
+      },
+
+      {
+        name: 'size',
+        type: '"md" | "lg"',
+        required: false,
+        description: 'md: 40px, lg: 48px',
+        defaultValue: 'md',
+      },
+    ]}
+  />
+);
+
+card(
+  <Example
+    id="basicExample"
+    name="Example"
+    defaultCode={`
+    function Example(props) {
+      const [item, setItem] = React.useState("");
+      const [selected, setSelected] = React.useState(null);
+
+      const options = Array.from(Array(20).keys()).map((item) => ({
+        value: "value-" + (item + 1),
+        label: "Label-" + (item + 1),
+      }));
+
+      const handleOnChange = ({ value }) => {
+        setItem(value);
+      };
+
+      const handleSelect = ({item}) => {
+        setSelected(item);
+      };
+
+      return (
+        <>
+          <Box marginBottom={2}>
+            <Text>Selected Item: {(selected && selected.label) || ""}</Text>
+          </Box>
+
+          <Typeahead
```

Comment:
@simkessy could we use `label` in this case to make it more accessible? That way, when someone clicks on the label, they focus into the typeahead + the label gets read out by screenreaders.

```
      const label = `Selected Item: ${(selected && selected.label) || ""}`;

      return (
        <Typeahead
          label={label}
          id="Typeahead-example"
          noResultText="No Results"
          data={options}
          placeholder="Select a Label"
          onChange={handleOnChange}
          value={item}
          onSelect={handleSelect}
        />
      );
```

Reply:
same with the example below

Reply:
![image](https://user-images.githubusercontent.com/5871660/87589251-ca9ffd00-c699-11ea-9401-d41472be7450.png)